phpBB/styles/prosilver/template/viewtopic_body.html

```diff
@@ -237,63 +255,35 @@ <h3 {% if postrow.S_FIRST_ROW %}class="first"{% endif %}>
 			<!-- IF $SHOW_POST_BUTTONS -->
 				<ul class="post-buttons">
 					<!-- EVENT viewtopic_body_post_buttons_before -->
-					<!-- IF postrow.U_EDIT -->
-						<li>
-							<a href="{postrow.U_EDIT}" title="{L_EDIT_POST}" class="button button-icon-only">
-								<i class="icon fa-pencil fa-fw" aria-hidden="true"></i><span class="sr-only">{L_BUTTON_EDIT}</span>
-							</a>
-						</li>
-					<!-- ENDIF -->
-					<!-- IF postrow.U_DELETE -->
-						<li>
-							<a href="{postrow.U_DELETE}" title="{L_DELETE_POST}" class="button button-icon-only">
-								<i class="icon fa-times fa-fw" aria-hidden="true"></i><span class="sr-only">{L_BUTTON_DELETE}</span>
-							</a>
-						</li>
-					<!-- ENDIF -->
-					<!-- IF postrow.U_REPORT -->
-						<li>
-							<a href="{postrow.U_REPORT}" title="{L_REPORT_POST}" class="button button-icon-only">
-								<i class="icon fa-exclamation fa-fw" aria-hidden="true"></i><span class="sr-only">{L_BUTTON_REPORT}</span>
-							</a>
-						</li>
-					<!-- ENDIF -->
-					<!-- IF postrow.U_WARN -->
-						<li>
-							<a href="{postrow.U_WARN}" title="{L_WARN_USER}" class="button button-icon-only">
-								<i class="icon fa-exclamation-triangle fa-fw" aria-hidden="true"></i><span class="sr-only">{L_BUTTON_WARN}</span>
-							</a>
-						</li>
-					<!-- ENDIF -->
-					<!-- IF postrow.U_INFO -->
-						<li>
-							<a href="{postrow.U_INFO}" title="{L_INFORMATION}" class="button button-icon-only">
-								<i class="icon fa-info fa-fw" aria-hidden="true"></i><span class="sr-only">{L_BUTTON_INFORMATION}</span>
-							</a>
-						</li>
-					<!-- ENDIF -->
-					<!-- IF postrow.U_QUOTE -->
-						<li>
-							<a href="{postrow.U_QUOTE}" title="{L_REPLY_WITH_QUOTE}" class="button button-icon-only">
-								<i class="icon fa-quote-left fa-fw" aria-hidden="true"></i><span class="sr-only">{L_BUTTON_QUOTE}</span>
-							</a>
-						</li>
-					<!-- ENDIF -->
+					<!-- INCLUDE posting_buttons.html -->
```

Comment:
What you include here, are the posting buttons, which are the BBCodes shown above the textarea when creating a post.
You include them again, in the dropdown version.

Reply:
hmm looks like all the changes I made to factor out all the JS responsive menu bulllshit got stripped out so ill have to fix all of this also.